app/src/main/java/com/simplecity/amp_library/ShuttleApplication.java

```diff
@@ -430,6 +433,20 @@ public void newUiEvent(UiEventType uiEventType){
         FirebaseIOUtils.saveUiEvent(uiEvent);
     }
 
```

Comment:
Can you please add Javadocs to this method? I think they exist in the OBA version?